spec.html

```diff
@@ -28562,10 +28567,10 @@ <h1>MakeDay ( _year_, _month_, _date_ )</h1>
           1. Let _y_ be ! ToInteger(_year_).
           1. Let _m_ be ! ToInteger(_month_).
           1. Let _dt_ be ! ToInteger(_date_).
-          1. Let _ym_ be _y_ + floor(_m_ / 12).
-          1. Let _mn_ be _m_ modulo 12.
+          1. Let _ym_ be _y_ + floor(_m_ / 12<sub>𝔽</sub>).
+          1. Let _mn_ be _m_ modulo 12<sub>𝔽</sub>.
           1. Find a value _t_ such that YearFromTime(_t_) is _ym_ and MonthFromTime(_t_) is _mn_ and DateFromTime(_t_) is 1; but if this is not possible (because some argument is out of range), return *NaN*.
-          1. Return Day(_t_) + _dt_ - 1.
+          1. Return Day(_t_) + _dt_ - 1<sub>𝔽</sub>.
```

Comment:
What's the order of operations here? It matters because Number arithmetic is not associative.